HelgaShiryaeva/assignment1/ReversedListAccessUtil

```diff
@@ -0,0 +1,91 @@
+public class ReversedListAccessUtil {
```

Comment:
From the design perspective it would be better if this was not a separate class but instead `MyList` provided a method to access its elements from the end. What do you think?